src/sidebar/services/frame-sync.ts

```diff
@@ -37,8 +37,14 @@ type DocumentInfo = {
  * JavaScript, it includes only the information needed to uniquely identify it
  * within the current session and anchor it in the document.
  */
-export function formatAnnot({ $tag, target, uri }: Annotation): AnnotationData {
+export function formatAnnot({
+  $cluster,
+  $tag,
+  target,
+  uri,
+}: Annotation): AnnotationData {
```

Comment:
It is currently the case that all the data we sent to the annotator for an annotation comes from the sidebar's internal `Annotation` object, but we're not obliged to keep things that way in future. We could for example look up additional information from other parts of the store and combine the results to generate the object sent to the annotator.

Reply:
Noted.

I think what I'd like to aim for is:

* There is a single annotation-object initialization point on both sides of the app (`annotator` and `sidebar`). This initialization logic may not have all of the answers — e.g. maybe we could initialize client-side properties to some default value — but I think there's value in having those properties at least initially set on the object in one place.
* We spend some time in the near future cleaning up some typing technical debt around `Annotation` — I'd like to more clearly separate API annotation typing from "app-wide" merged client-side properties versus what a constructed annotation object looks like in the `annotator` vs. what an annotation object looks like in the `sidebar` (these things are all different in reality). We might opt to think about changing the naming or structure of client-side properties, as you have suggested, at that time. That will help clarify this situation, I believe.

One future use case we've briefly discussed is having the sidebar's annotation cards have visual continuity with the annotation's cluster, e.g. making the highlight/quote area echo the color of the associated cluster. For that, the sidebar will need `$cluster` information, so I would like to leave the `$cluster` logic in the `annotations` store module _for now_ but will keep this all in mind...